src/components/DefaultEpic.tsx

```diff
@@ -56,51 +61,70 @@ const imageStyles = css`
-            </p>
-            <p className={bodyStyles}>
-                The Guardian will engage with the most critical issues of our time – from the
-                escalating climate catastrophe to widespread inequality to the influence of big tech
-                on our lives. At a time when factual information is a necessity, we believe that
-                each of us, around the world, deserves access to accurate reporting with integrity
-                at its heart.
-            </p>
-            <p className={bodyStyles}>
-                Our editorial independence means we set our own agenda and voice our own opinions.
-                Guardian journalism is free from commercial and political bias and not influenced by
-                billionaire owners or shareholders. This means we can give a voice to those less
-                heard, explore where others turn away, and rigorously challenge those in power.
-            </p>
-            <p className={bodyStyles}>
-                We hope you will consider supporting us today. We need your support to keep
-                delivering quality journalism that’s open and independent. Every reader
-                contribution, however big or small, is so valuable.{' '}
-                <strong className={highlightWrapperStyles}>
-                    <span className={highlightStyles}>
-                        Support The Guardian from as little as £1 - and it only takes a minute.
-                        Thank you.
-                    </span>
-                </strong>
-            </p>
-            <div className={buttonWrapperStyles}>
-                <CallToAction
-                    url="https://support.theguardian.com/uk/contribute"
-                    linkText="Support The Guardian"
-                />
-                <img
-                    src="https://assets.guim.co.uk/images/acquisitions/2db3a266287f452355b68d4240df8087/payment-methods.png"
-                    alt="Accepted payment methods: Visa, Mastercard, American Express and PayPal"
-                    className={imageStyles}
-                />
-            </div>
-        </section>
-    );
+type Props = {
+    heading: string | undefined;
+    paragraphs: string[];
+    highlighted: string[];
+};
+
+type HighlightedProps = {
+    highlighted: string[];
+};
+
+type BodyProps = {
+    paragraphs: string[];
+    highlighted: string[];
 };
+
+const Highlighted: React.FC<HighlightedProps> = ({ highlighted }: HighlightedProps) => (
+    <strong className={highlightWrapperStyles}>
+        {' '}
+        {highlighted.map((highlightText, idx) => (
+            <span
+                key={idx}
+                className={highlightStyles}
+                dangerouslySetInnerHTML={{ __html: replacePlaceholders(highlightText) }}
+            />
+        ))}
+    </strong>
+);
+
+const Body: React.FC<BodyProps> = ({ highlighted, paragraphs }: BodyProps) => (
+    <>
+        {paragraphs.length > 0 &&
```

Comment:
Don't think we need to check for the length of the array before we map through it, do we? With a length of `0` the map callback would be called `0` times, right?

Reply:
Ah thanks, good spot! I think at some point I had a version where I _did_ want this, but definitely not needed now.